src/custom/components/WalletModal/WalletModalMod.tsx

```diff
@@ -211,13 +212,21 @@ export default function WalletModal({
     }
 
     connector &&
-      activate(connector, undefined, true).catch((error) => {
-        if (error instanceof UnsupportedChainIdError) {
-          activate(connector) // a little janky...can't use setError because the connector isn't set
-        } else {
-          setPendingError(true)
-        }
-      })
+      activate(connector, undefined, true)
+        .catch((error) => {
+          if (error instanceof UnsupportedChainIdError) {
+            activate(connector) // a little janky...can't use setError because the connector isn't set
+          } else {
+            setPendingError(true)
+          }
+        })
+        .then(() => {
+          const tmpWCdata = localStorage.getItem(WALLET_CONNECT_STORAGE_KEY)
+
+          if (tmpWCdata) {
+            setTimeout(() => localStorage.setItem(WALLET_CONNECT_STORAGE_KEY, tmpWCdata), 1500)
```

Comment:
not in love with this but it does cover the issue.

i would open an issue with them and ask about:
1. a constructor param or sth allowing to set whether we want this storage wipe feature
2. them to fix their listeners .on('Web3ReactUpdate') to allow us to subscribe to changes there